app/src/main/java/org/mozilla/reference/browser/browser/BaseBrowserFragment.kt

```diff
@@ -402,28 +363,21 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Activit
                 view = view,
             )
         }
-
-        val composeView = view.findViewById<ComposeView>(R.id.compose_view)
-        if (shouldUseComposeUI) {
-            composeView.visibility = View.VISIBLE
-            composeView.setContent { BrowserToolbar() }
-
-            val params = swipeRefresh.layoutParams as CoordinatorLayout.LayoutParams
-            params.topMargin = resources.getDimensionPixelSize(R.dimen.browser_toolbar_height)
-            swipeRefresh.layoutParams = params
-        }
     }
 
     private fun fullScreenChanged(enabled: Boolean) {
         if (enabled) {
             activity?.enterImmersiveMode()
-            toolbar.visibility = View.GONE
             engineView.setDynamicToolbarMaxHeight(0)
         } else {
             activity?.exitImmersiveMode()
-            toolbar.visibility = View.VISIBLE
             engineView.setDynamicToolbarMaxHeight(resources.getDimensionPixelSize(R.dimen.browser_toolbar_height))
         }
+        parentFragment?.parentFragmentManager
```

Comment:
Communicating between fragments or with the parent [should normally be done through a parent ViewModel](https://developer.android.com/guide/fragments/communicate) or in our case a Store.
Or as it happens in many cases and if we know that our parent is showing a toolbar we could just access it directly.

Curious why going with the fragment result instead.